frontend/src/components/metadata/EditMetadataDefinitionModal.tsx

```diff
@@ -0,0 +1,671 @@
+												<ClowderInput
+													margin="normal"
+													fullWidth
+													required
+													id="metadata-context"
+													label="Term"
+													placeholder="Please enter context term"
+													value={item["term"]}
+													sx={{
+														mt: 1,
+														mr: 1,
+														alignItems: "right",
+														width: "300px",
+													}}
+													onChange={(event) => {
+														updateContext(idx, "term", event.target.value);
+													}}
+												/>
+											</Grid>
+											<Grid item>
+												<Autocomplete
+													id="metadata-auto-complete"
+													freeSolo
+													autoHighlight
+													inputValue={item["iri"]}
+													options={contextUrlMap["frequently_used"].map(
+														(option) => option.url
+													)}
+													onInputChange={(event, value) => {
+														updateContext(idx, "iri", value);
+													}}
+													renderInput={(params) => (
+														<ClowderInput
+															{...params}
+															required
+															label="IRI"
+															fullWidth
+														/>
+													)}
+													sx={{
+														mt: 1,
+														ml: 2,
+														alignItems: "right",
+														minWidth: "20em",
+													}}
+												/>
+											</Grid>
+											<IconButton
+												color="primary"
+												size="small"
+												onClick={() => addNewContext(idx)}
+											>
+												<AddBoxIcon />
+											</IconButton>
+											{idx == 0 ? (
+												<></>
+											) : (
+												<IconButton
+													color="primary"
+													size="small"
+													onClick={() => removeContext(idx)}
+												>
+													<DeleteOutlineIcon />
+												</IconButton>
+											)}
+										</Grid>
+									);
+								})}
+
+								<Button
+									variant="contained"
+									onClick={() => validateFormData(activeStep)}
+								>
+									Next
+								</Button>
+							</form>
+						</StepContent>
+					</Step>
+
+					{formInput.fields.map((input, idx) => {
+						return (
+							<Step key={idx}>
+								{idx == 0 ? (
+									<StepButton
+										color="inherit"
+										onClick={() => {
+											setActiveStep(idx + 1);
+										}}
+									>
+										<StepLabel>
+											Edit metadata entry*
+											{idx == activeStep - 1 ? (
+												<IconButton
+													color="primary"
+													size="small"
+													onClick={() => addNewField(idx)}
+												>
+													<AddBoxIcon />
+												</IconButton>
+											) : (
+												<></>
+											)}
+										</StepLabel>
+									</StepButton>
+								) : (
+									<StepButton
+										color="inherit"
+										onClick={() => {
+											setActiveStep(idx + 1);
+										}}
+									>
+										<StepLabel>
+											Add additional entry
+											{idx == activeStep - 1 ? (
```

Comment:
Once click add metadata entry and save updated; the next time when editing, it is still showing "add additional entry"
<img width="814" alt="image" src="https://github.com/clowder-framework/clowder2/assets/13950475/f6a39461-4448-43f4-9277-7b813f1949ab">

Reply:
I was thinking users may want to 'add additional entry' in addition to updating the existing ones

Reply:
My bad. I got what you meant. Resolved.  For simplicity, I just put 'Edit metadata entry' for now. Deleting metadata entry in btw the list and then tracking each of them seemed to me overkill for now. If needed, I can address that.

Reply:
looks correct now.